WcaOnRails/db/migrate/20170624115851_add_wca_financial_committee.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class AddWcaFinancialCommittee < ActiveRecord::Migration[5.0]
+  def change
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, email, rank, created_at, updated_at) values ('wfc', 'finance@worldcubeassociation.org', 5, NOW(), NOW());
+    SQL
+
+    execute <<-SQL
+      UPDATE teams set rank = 3 where friendly_id = 'wdc';
+    SQL
+
+    execute <<-SQL
+      UPDATE teams set rank = 7 where friendly_id = 'wrc';
+    SQL
+
+    execute <<-SQL
+      UPDATE teams set rank = 9 where friendly_id = 'wrt';
+    SQL
+
+    execute <<-SQL
+      UPDATE teams set rank = 11 where friendly_id = 'wst';
+    SQL
+
+    execute <<-SQL
+      INSERT INTO team_members (team_id, user_id, start_date, team_leader, created_at, updated_at) values (6, 271, '2017-06-14', 1, NOW(), NOW())
```

Comment:
I don't like this, because it assumes the team_id of the WFC is 6 (also, the user_ids are pretty arbitrary). Maybe it would be better to do this stuff via the manage team ui after the migration is run? If a developer is running with a local database, these users may not even exist.